### PR TITLE
chore: setup a11y tools

### PR DESCRIPTION
**Pull request by andreapiras**
☝️ make the title meaningful and follow conventional commits standards

## 📎 Jira ticket

https://transferwise.atlassian.net/browse/DS-440

## ❓ Context  

In order to meet accessibility requirements, some tools have been added along some documentation on how to contribute 

## 🚀 Changes 
1) Clean up old eslint rules
2) Add react-axe to dev and docs ev
3) Run eslint on test with jest-eslint-run
4) Add contribution guidelines

## ✅ Checklist

- [ ] All changes are covered by tests
- [ ] All changes have been crossbrowser checked, especially IE11
- [ ] The changes are covered in docs

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -45,10 +45,6 @@ aliases:
     name: Publish packages
     command: npx lerna publish -m "[skip ci] Release" --yes
 
-  - &lint
```
Comment (andreapiras) [inline .circleci/config.yml:48]: Remove eslint. Now it runs via jest-eslint-runner as part of the test

File: .eslintrc [hunk 2]
```diff
@@ -3,70 +3,31 @@
 {
   "extends": "@transferwise",
   "rules": {
-    "no-mixed-operators": 0,
```
Comment (andreapiras) [inline .eslintrc:6]: Removed old unused rules

File: packages/components/.storybook/config.js [hunk 3]
```diff
@@ -18,9 +18,18 @@ const style = {
   width: '100vw',
 };
 
-const StrictModeDecorator = storyFn => <StrictMode>{storyFn()}</StrictMode>;
+if (process.env.NODE_ENV !== 'production') {
+  const ReactDOM = require('react-dom');
```
Comment (andreapiras) [inline packages/components/.storybook/config.js:22]: Add react-axe to storybook console

File: packages/components/.storybook/config.js [hunk 4]
```diff
@@ -18,9 +18,18 @@ const style = {
...
+  var axe = require('react-axe');
+  var context = {
+    include: [['#story-root']],
```
Comment (andreapiras) [inline packages/components/.storybook/config.js:25]: node to check for a11y

File: packages/components/src/phoneNumberInput/PhoneNumberInput.story.js [hunk 5]
```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import PhoneNumberInput from './PhoneNumberInput';
 import { select, boolean } from '@storybook/addon-knobs';
+import { action } from '@storybook/addon-actions';
```
Comment (andreapiras) [inline packages/components/src/phoneNumberInput/PhoneNumberInput.story.js:4]: Removed console causing no-aler eslint warning

File: .eslintrc [hunk 6]
```diff
@@ -3,70 +3,31 @@
...
-      {
-        "required": {
-          "some": [
-            "nesting",
-            "id"
-          ]
-        }
-      }
-    ],
+    "react/destructuring-assignment": [0, "never"],
```
Comment (helendurranttw) [inline .eslintrc:6]: Why did you turn this on?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 7]
```diff
@@ -2,18 +2,15 @@
...
-
 ## 💬 Considerations <!-- additional info for reviewing (e.g links to Mockups, docs etc.), discussion topics -->
 
-
 ## ✅ Checklist
 
 - [ ] All changes are covered by tests
 - [ ] All changes have been crossbrowser checked, especially IE11
 - [ ] The changes are covered in docs
+- [ ] Accessibility checks have been performed. Console doesn't contain any violations (Accessibility checks)[https://github.com/transferwise/neptune-web/blob/master/packages/components/CONTRIBUTING.md#accessibility]
```
Comment (helendurranttw) [inline .github/PULL_REQUEST_TEMPLATE.md:16]: Nice 👍

File: packages/components/.storybook/config.js (hunk 4 above)
Comment (helendurranttw) [reply packages/components/.storybook/config.js:25]: Use `const` here

File: packages/docs/pages/_app.js [hunk 8]
```diff
@@ -10,6 +10,12 @@ import '@transferwise/icons/dist/icons.min.css';
 
 import '../static/assets/main.css';
 
+if (process.env.NODE_ENV !== 'production' && typeof window !== 'undefined') {
+  const ReactDOM = require('react-dom');
+  var axe = require('react-axe');
```
Comment (helendurranttw) [inline packages/docs/pages/_app.js:15]: `const`

File: packages/components/CONTRIBUTING.md [hunk 9]
```diff
@@ -19,6 +19,24 @@ Please use [react-testing-library](https://github.com/testing-library/react-test
 Historically we've used Enzyme so most of tests built using it's API but we encourage you to use react-testing-library when creating a new component
 also, feel free to update tests of existing components (e.g when you do some changes) and flag about this during code review.
 
+# Accessibility
```
Comment (helendurranttw) [inline packages/components/CONTRIBUTING.md:22]: Really nice. A few minor typos, I'll fix them up.

File: .eslintrc (hunk 6 above)
Comment (andreapiras) [reply .eslintrc:6]: It's the same rule as old line 7, It's so showing cause it has been reformatted

File: packages/components/CONTRIBUTING.md [hunk 10]
```diff
@@ -19,6 +19,34 @@ Please use [react-testing-library](https://github.com/testing-library/react-test
...
+
+A short series of manual checks are also necessary. For all changes, perform the following:
+
+## Keyboard tests
+
+1. Component has been tested to be keyboard accessible: i.e. it is fully functional using only the keyboard
+2. Keyboard focus order matches the visual layout
+3. Ensure that controls have visible focus states
+
+## Custom browser settings
```
Comment (helendurranttw) [inline packages/components/CONTRIBUTING.md:36]: Let's rename this "Zoom settings"

**Pull request merged**